benchmark/LICENSE.md

```diff
@@ -0,0 +1,24 @@
+The files benchmark/pprinthelper.jl, benchmark/pprintjudge.jl,
+benchmark/pprintresult.jl, benchmark/runbenchmarks.jl, and benchmark/runjudge.jl
+contain code copied from the Transducers.jl package
+(https://github.com/tkf/Transducers.jl/) which is licensed under the MIT License:
+
+>Copyright (c) 2018 Takafumi Arakaki
+>
+>Permission is hereby granted, free of charge, to any person obtaining a copy
+>of this software and associated documentation files (the "Software"), to deal
+>in the Software without restriction, including without limitation the rights
+>to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+>copies of the Software, and to permit persons to whom the Software is
+>furnished to do so, subject to the following conditions:
+>
+>The above copyright notice and this permission notice shall be included in all
+>copies or substantial portions of the Software.
+>
+>THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+>IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+>FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+>AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+>LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+>OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+>SOFTWARE.
```

Comment:
It's not really obvious that code like:
```julia
# This file was copied from Transducers.jl
# which is available under an MIT license (see LICENSE).
using PkgBenchmark
benchmarkpkg(
    dirname(@__DIR__),
    BenchmarkConfig(env = Dict("JULIA_NUM_THREADS" => "1")),
    resultfile = joinpath(@__DIR__, "result.json"),
```
is justified to declare the MIT license. This is just usage of a package, unrelated to Transducers?